Add bert-base-uncased-ldc2012t13_pfeiffer

vblagoje opened the pull request:
Adds a POS Tagger trained on LDC2012T13 from https://github.com/UniversalDependencies/UD_English-EWT/ Trained adapter has ~.95 accuracy on test dataset. 

A few things remain to be resolved:

1. I didn't know which organization to use so I just added the adapter to ukp
2. Test colab notebook is [here](https://colab.research.google.com/drive/1KoRQSQVOw-KaCOA1Q3vlQz10YcDguzTs)
3. The notebook has a basic inferencing version and TokenClassificationPipeline version
4. As you know the notebook should be tested by loading adapter from the local filesystem
5. TokenClassificationPipeline does not work until adapter_names is passed to the [model](https://github.com/Adapter-Hub/adapter-transformers/blob/9118a50662d2e7e09817da90bb6b194c604f0c18/src/transformers/pipelines.py#L915)
6. During inferencing, the model returns CLS and SEP tokens as well. See the notebook for details. Is that expected?

vblagoje committed:
Add bert-base-uncased-ldc2012t13_pfeiffer

calpt commented:
Hi @vblagoje,

Thanks for your contribution! Looks very good overall. (Please just have another look at the formatting checks for the subtask yaml.)

Regarding your issues:
1 You can just create a new subfolder e.g. with your GitHub username and put the adapter there. Then you can collect all your contributions in a single place (in case you add more adapters in the future)

2 Great, maybe add a link to this notebook as well as the final score on the test set to the `description` field?

4 just a side note: you can also load the adapter directly from the url you provided

5 Thanks for mentioning this. We will shortly add an update (Adapter-Hub/adapter-transformers#44) that adds the method `model.set_active_adapters(['pos'])` to every model class. After calling this method after loading the adapter, passing `adapter_names` in every forward pass will not be needed anymore.

6 Seems to be the expected result as implemented by HuggingFace Transformers.

vblagoje commented:
Hey @calpt ,

Great feedback. I fixed the formatting issues. In addition, I also changed the org for the adapter, made description a bit more verbose, added F1 score, and example notebook link. I'll make the notebook a bit more user-friendly next week. Thanks for the pointers under items 4,5, and 6. LMK if there are any additional adjustments needed.  Cheers.

calpt reviewed (approved):
Looks good 👍
